pages/index.vue

```diff
@@ -119,12 +123,54 @@
 </template>
 
 <style>
-.get-involved-button img {
-  width: 80%;
-  object-fit: contain;
+.get-involved-button {
```

Comment:
we should probably move these styles to a CSS file of some sort at some point :)